exercises/patterns/printMultiplicationTable/printMultiplicationTable.js

```diff
@@ -21,6 +21,14 @@ let helpers = require('../printHelpers');
  * @param {number} n - The size of the multiplication table to print
  */
 function printMultiplicationTable(n) {
+  let columnWidth = (n**2).toString().length;
+  for(let i = 1; i < n + 1; i ++){
+    for(let j = 1; j < n + 1; j ++){
+      helpers.printCountTimes((i * j).toString().padStart(columnWidth, " "), 1)
```

Comment:
Nice solution, you can also use `helpers.printCountTimes("\t", 2)` to print tab spaces.